mkdocs.yml

```diff
@@ -65,6 +65,7 @@ nav:
       - explanations/index.md
       - About this documentation: explanations/about-this-documentation.md
       - About Docker and Docker Compose: explanations/about-docker-and-docker-compose.md
+      - About Domain-Driven Design: explanations/about-ddd.md
```

Comment:
As the title of the page is "About Domain-driven design (DDD)", I would rename this as `- About Domain-driven design (DDD): explanations/about-domain-driven-design.md`.